blockchainetl/streaming/mysql_utils.py

```diff
@@ -0,0 +1,7 @@
+from sqlalchemy.dialects.postgresql import insert
```

Comment:
the insert is from PostgreSQL ?

Reply:
Oh, it is my mistake, it should be from MySQL, I will re-create a PR later.

Reply:
you don't need to re-create PR. just push new commits